pandas-stubs/core/groupby/generic.pyi

```diff
@@ -168,7 +168,7 @@ class SeriesGroupBy(GroupBy[Series[S1]], Generic[S1, ByT]):
         self,
     ) -> Iterator[tuple[ByT, Series[S1]]]: ...
 
-_TT = TypeVar("_TT", bound=Literal[True, False])
+_TT = TypeVar("_TT", bound=Literal[True, False], default=Literal[True])
```

Comment:
I'm a bit surprised that you didn't have to change the `import` of `TypeVar` to come from `typing_extensions` here since you are using the `default` feature.